packages/nestjs/src/bull-board.root-module.ts

```diff
@@ -17,8 +17,13 @@ export class BullBoardRootModule implements NestModule {
   }
 
   configure(consumer: MiddlewareConsumer): any {
-    const globalPrefix = this.applicationConfig.getGlobalPrefix() || '';
-    this.adapter.setBasePath(`${ globalPrefix }${ this.options.route }`);
+    const globalPrefix = this.applicationConfig.getGlobalPrefix();
+    const prefix = `${globalPrefix}${this.options.route}`.replace(
+      /^\/?([^\/]+(?:\/[^\/]+)*)\/?$/,
```

Comment:
Can you explain this REGEX?

Reply:
- A forward slash is added to the beginning of the path if it’s not already there.
- Empty strings are handled correctly.
- Trailing slashes are excluded.

Maybe we don't need to exclude the trailing slash of the route and simplify the regex pattern to:
```js
    const prefix = `${globalPrefix}${this.options.route}`.replace(/^(?!\/|$)/, '/');
```

https://jsfiddle.net/dimbslmh/51xpqgwa/

Reply:
So, maybe, insead of cryptic regex, we can use `startsWith` && `endsWith`  string method to adjust these trailing slashes.

WDYT?

Reply:
Sure, like this?
```js
    const addForwardSlash = (path) => path.startsWith('/') || path === '' ? path : `/${path}`;
    const prefix = addForwardSlash(globalPrefix + this.options.route);
```
https://jsfiddle.net/dimbslmh/vjhzp5ao/3/

Reply:
I've synced the master branch code and updated the PR.